### PR TITLE
remove conflicx

### PR DESCRIPTION
**Pull request by lylylong**

### CODE REVIEW

**Review by Shamakhan786 — APPROVED** [review]

**Review by BrieBencich — APPROVED** [review]
Looks greats

File: index.html [hunk 1]
```diff
@@ -210,44 +209,3 @@ <h2 id="country-province" class="ml-5 mt-3"></h2>
     <script src="./assets/js/script.js"></script>
   </body>
 </html>
-=======
```
Comment (lylylong) [inline index.html:213]: The following codes should be removed

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 ## Website URL
 
-TBA
+https://briebencich.github.io/Covid-19-Tracking/#Covid-19-Tests
```
Comment (lylylong) [inline README.md:29]: This URL should replace the TBA above

File: index.html [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <!DOCTYPE html>
```
Comment (lylylong) [inline index.html:1]: Remove this section

**Pull request merged**